tests/integration/suite/actors/reminders/migration/scheduler/distributed.go

```diff
@@ -0,0 +1,164 @@
+	}
+}
+
+func (d *distributed) Run(t *testing.T, ctx context.Context) {
+	schedConfig := `apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: schedulerreminders
+spec:
+  features:
+  - name: SchedulerReminders
+    enabled: true
+`
+
+	optsApp1 := []daprd.Option{
+		daprd.WithResourceFiles(d.db.GetComponent(t)),
+		daprd.WithPlacementAddresses(d.place.Address()),
+		daprd.WithSchedulerAddresses(d.scheduler.Address()),
+		daprd.WithAppPort(d.app1.Port()),
+	}
+	optsApp2 := []daprd.Option{
+		daprd.WithResourceFiles(d.db.GetComponent(t)),
+		daprd.WithPlacementAddresses(d.place.Address()),
+		daprd.WithSchedulerAddresses(d.scheduler.Address()),
+		daprd.WithAppPort(d.app2.Port()),
+		daprd.WithConfigManifests(t, schedConfig),
+	}
+	optsApp1WithScheduler := []daprd.Option{
+		daprd.WithResourceFiles(d.db.GetComponent(t)),
+		daprd.WithPlacementAddresses(d.place.Address()),
+		daprd.WithSchedulerAddresses(d.scheduler.Address()),
+		daprd.WithAppPort(d.app1.Port()),
+		daprd.WithConfigManifests(t, schedConfig),
+	}
+
+	daprd1 := daprd.New(t, optsApp1...)
+	daprd2 := daprd.New(t, optsApp2...)
+	daprd3 := daprd.New(t, optsApp2...)
+	daprd4 := daprd.New(t, optsApp2...)
+	daprd5 := daprd.New(t, optsApp2...)
+	daprd6 := daprd.New(t, optsApp1WithScheduler...)
+
+	daprd1.Run(t, ctx)
+	daprd1.WaitUntilRunning(t, ctx)
+	client := daprd1.GRPCClient(t, ctx)
+
+	var wg sync.WaitGroup
+	wg.Add(200)
+	for i := range 100 {
+		go func(i int) {
+			defer wg.Done()
+			_, err := client.RegisterActorReminder(ctx, &rtv1.RegisterActorReminderRequest{
+				ActorType: "myactortype",
+				ActorId:   strconv.Itoa(i),
+				Name:      strconv.Itoa(i),
+				DueTime:   "10000s",
+			})
+			assert.NoError(t, err)
+		}(i)
+		go func(i int) {
+			defer wg.Done()
+			_, err := client.RegisterActorReminder(ctx, &rtv1.RegisterActorReminderRequest{
+				ActorType: "myactortype2",
+				ActorId:   strconv.Itoa(i),
+				Name:      strconv.Itoa(i),
+				DueTime:   "10000s",
+			})
+			assert.NoError(t, err)
+		}(i)
+	}
+	wg.Wait()
+
+	assert.Len(t, d.db.ActorReminders(t, ctx, "myactortype").Reminders, 100)
+	assert.Len(t, d.db.ActorReminders(t, ctx, "myactortype2").Reminders, 100)
+	assert.Empty(t, d.scheduler.EtcdJobs(t, ctx))
+
+	daprd1.Cleanup(t)
+	daprd2.Run(t, ctx)
+	daprd3.Run(t, ctx)
+	daprd4.Run(t, ctx)
+	daprd5.Run(t, ctx)
+	daprd2.WaitUntilRunning(t, ctx)
+	daprd3.WaitUntilRunning(t, ctx)
+	daprd4.WaitUntilRunning(t, ctx)
+	daprd5.WaitUntilRunning(t, ctx)
+
+	assert.Len(t, d.db.ActorReminders(t, ctx, "myactortype").Reminders, 100)
+	assert.Len(t, d.db.ActorReminders(t, ctx, "myactortype2").Reminders, 100)
+	assert.Len(t, d.scheduler.EtcdJobs(t, ctx), 100)
+
+	daprd6.Run(t, ctx)
+	daprd6.WaitUntilRunning(t, ctx)
+
+	assert.Len(t, d.db.ActorReminders(t, ctx, "myactortype").Reminders, 100)
+	assert.Len(t, d.db.ActorReminders(t, ctx, "myactortype2").Reminders, 100)
+	assert.Len(t, d.scheduler.EtcdJobs(t, ctx), 200)
+
+	daprd2.Cleanup(t)
```

Comment:
should these be deferred in case the test runs into an err?

Reply:
Yeah I agree, thanks @cicoyle